app/presenters/service_manual_guide_presenter.rb

```diff
@@ -13,8 +13,12 @@ def initialize(content_item)
   end
 
   def content_owner
-    content_owner_data = content_item["details"]["content_owner"]
-    ContentOwner.new(content_owner_data["title"], content_owner_data["href"])
+    content_owner_data = Array(content_item["links"].try(:[], "content_owners")).first
```

Comment:
`Array(...).first` here is also important :) I could rewrite it as 
```
Array(content_item["links"] && content_item["links"]["content_owners"]).first
``` 
but I don't find it any more readable :confused: 

http://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig would be ideal here :)